### PR TITLE
Adjust `linea_estimateGas` reference page for Mainnet activation

### PR DESCRIPTION
**Pull request by jlwllmr**
`linea_estimateGas` is available on mainnet as of Alpha v3.4 (30th July). Edits existing reference page to reflect this.

### CODE REVIEW

File: docs/developers/reference/api/linea-estimategas.mdx [hunk 1]
```diff
@@ -11,12 +11,9 @@ import TabItem from '@theme/TabItem';
...
 
-`linea_estimateGas` is currently not available on the Linea Sepolia testnet, but will be activated
-soon. **Mainnet activation is scheduled for July 30, 2024.** 
-
-It will only be fully compatible with endpoints using the Besu client — if it is unavailable, try 
-updating Besu. If you prefer to use Geth, `linea_estimateGas` is only available if you redirect
-your request through an Infura endpoint.
+The `linea_estimateGas` is only fully compatible with endpoints using the Besu client — if it is 
+unavailable, try updating Besu. If you prefer to use Geth, `linea_estimateGas` is only available if 
+you redirect your request through an Infura endpoint.
```
Comment (bgravenorst) [inline docs/developers/reference/api/linea-estimategas.mdx:16]: We may need to refine this statement. For example, I don't think this method is available in the vanilla Besu client (Used in our Run a node instructions). I think it's only when running [linea-sequencer](https://github.com/Consensys/linea-sequencer?tab=readme-ov-file#quickstart---running-linea-besu-with-plugins).

File: docs/developers/reference/api/linea-estimategas.mdx [hunk 2]
```diff
@@ -75,10 +72,10 @@ You can also call the API using [Infura's supported Linea endpoints](https://doc
   <TabItem value="cURL request">
 
    ```bash
-  curl https://rpc.sepolia.linea.build \
+    curl https://rpc.sepolia.linea.build \
```
Comment (bgravenorst) [inline docs/developers/reference/api/linea-estimategas.mdx:75]: Perhaps update this to mainnet?

File: docs/developers/reference/api/linea-estimategas.mdx [hunk 3]
```diff
@@ -97,3 +94,27 @@ You can also call the API using [Infura's supported Linea endpoints](https://doc
 ```
   </TabItem>
 </Tabs>
+
+## Usage
```
Comment (bgravenorst) [inline docs/developers/reference/api/linea-estimategas.mdx:98]: I think this should go into Example section as a new tab called "ethers.js"

**Review by bgravenorst — CHANGES_REQUESTED** [review]

**Comment by bgravenorst** [comment]
As an aside, the API method names are not displaying correctly, the backticks are displaying in the menu.

Screenshot 2024-07-31 at 8 33 39 AM

File: docs/developers/reference/api/linea-estimategas.mdx (hunk 1 above)
Comment (jlwllmr) [reply docs/developers/reference/api/linea-estimategas.mdx]: Good point. I have a ticket open to make these edits to the run a node instructions — will confirm with the team then update the run a node instructions and this para.

**Review by bgravenorst — APPROVED** [review]
lgtm

**Pull request merged**